### PR TITLE
[K32W] Save PDM to external flash on the DK6 board

### PR DESCRIPTION
**Pull request by doru91**
Problem
What is being fixed?

Decrease the usage of the internal flash space (640K).

Change overview
Move Persistent Data Manager (PDM) from the internal flash to the external flash available on the DK6 development board.
Basically, we saved 32.5K of internal flash.  

Testing
Created a large application (almost 640K) and run it ok. Before, it was impossible to run it because only PDM required 32.5K of the internal flash.

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

**Review by woody-apple — CHANGES_REQUESTED** [review]
Can you please fill out the PR template? Thanks!

**Comment by doru91** [comment]
> Can you please fill out the PR template? Thanks!

Done.

**Review by jmartinez-silabs — APPROVED** [review]

File: examples/platform/k32w/app/ldscripts/chip-k32w061-linker.ld [hunk 1]
```diff
@@ -328,5 +320,6 @@ SECTIONS
     PROVIDE(Chip_LOWPOWER_ChipSoftwareReset = 0x03003fa1);
 
     __StackLimit = _vStackTop - STACK_SIZE;
-    ASSERT(__StackLimit >= _end_boot_resume_stack, "Possible stack corruption with data/bss/boot_stack")
+    /* ASSERT(__StackLimit >= _end_boot_resume_stack, "Possible stack corruption with data/bss/boot_stack") */
```
Comment (LuDuda) [inline examples/platform/k32w/app/ldscripts/chip-k32w061-linker.ld:323]: nit: remove commented code?

**Review by LuDuda — APPROVED** [review]

File: examples/platform/k32w/app/ldscripts/chip-k32w061-linker.ld (hunk 1 above)
Comment (doru91) [reply examples/platform/k32w/app/ldscripts/chip-k32w061-linker.ld]: thanks, @LuDuda . I refactored this code a little.

**Review by woody-apple — APPROVED** [review]

**Pull request merged**